examples/UI/CampaignBudgetsCDN/scripts/controllers.js

```diff
@@ -84,11 +98,25 @@ angular.module('budgetsApp')
 
 
             $scope.submitBudget = function (Budget) {
+<<<<<<< HEAD
```

Comment:
Those conflicts need to be resolved, the block of code between
+<<<<<<<< HEAD

+=========

is your version 

between 

+=========


+>>>>>>>>>> upstream/master

is upstream version